awot/io/read_sonde.py

```diff
@@ -120,44 +127,187 @@ def read_dropsonde_data(filePath, split_file=True):
     data = dict()
     data['metadata'] = header
     data['temperature'] = _build_dict(T,
-        'c', 'Temperature of ambient air', 'Temperature')
+                                      'c', 'Temperature of ambient air',
+                                      'Temperature')
     data['dewpoint'] = _build_dict(TD,
-        'c', 'Dewpoint temperature of ambient air', 'Dewpoint Temperature')
+                                   'c', 'Dewpoint temperature of ambient air',
+                                   'Dewpoint Temperature')
     data['presssure'] = _build_dict(P, 'hPa',
-        'Pressure of ambient air', 'Pressure')
+                                    'Pressure of ambient air', 'Pressure')
     data['relative_humidity'] = _build_dict(RH,
-        '%', 'Relative Humidity of ambient air’, ‘Relative Humidity')
+                                            '%', 'Relative Humidity of ambient air’,
+                                            'Relative Humidity')
     data['u_component'] = _build_dict(Uwind,
-        'm/s', 'u component of wind', 'U component')
+                                      'm/s', 'u component of wind',
+                                      'U component')
     data['v_component'] = _build_dict(Vwind,
-        'm/s', 'v component of wind', 'V component')
+                                      'm/s', 'v component of wind',
+                                      'V component')
     data['height'] = _build_dict(H,
-        'm', 'Geometric Height in meters', 'Height')
+                                 'm', 'Geometric Height in meters',
+                                 'Height')
     data['data_format'] = 'dropsonde'
 
     return data
 
 
-# splits a cls file for use in the programm(needs fixing)
-def split_cls_file(filename):
-
-    splitString1 = 'Data Type:'
-    splitString2 = 'Project ID:'
-    splitString3 = '------'
-
-    f = open(filename,'r')
-    newString = ''
-    splitFileList = []
-    f.readline()
-    s =0
-    x=0
-    for line in f:
-        if(splitString1 or splitString2) not in line:
-            newString = newString + line
-        else:
-            splitFileList.append(newString)
-            newString = ''
-
-    splitFileList.append(newString)
+def find_headers(filename):
+    '''
+    Finds the location of the headers of each data instance inside the CLS File
+    Inputs:
+    file path
+    #============
+    output:
+    #list of header locations and number of headers
+    '''

```

Comment:
Update docstring to reflect changes in variables names. Can put the output under a note, use Parameters instead of Inputs as with rest of package. Trying to stay consistent.